### ISSUE TITLE
Radar and Reccommended blogs are still showing despite disabling them in tweaks

### ISSUE THREAD

**Issue by nautiscarader** [new-xkit/XKit#1859]
Radar and Reccommended blogs are still showing despite disabling them in tweaks. God,the new dsahboard sucks.

**Comment by AprilSylph** [comment]
Tweaks has not been updated for the new dashboard yet. Please see #1802

**Comment by nautiscarader** [comment]
Well, then please update them, cos it seems the new dashboards is permanent now.

**Comment by nightpool** [comment]
tweaks is such a big extension, it's good to have individual issues to track individual pieces we want to update

**Comment by nautiscarader** [comment]
Avatars are also still sliding up and down, despite me disabling that.

**Comment by invalidCards** [comment]
@nautiscarader are you still experiencing these issues? The latest version of Tweaks should see both of these fixed (to some extent).

**Comment by nautiscarader** [comment]
No, Avatars stay in place as they should.

**Comment by nautiscarader** [comment]
So yeah, nothing is fixed :/

**Comment by lihatt** [comment]
I see the same behavior as nautiscarader.

**Comment by lihatt** [comment]
Received update to 6.0.9 just now. Hiding Recommended Blogs seems to be working now, and actually sticking? I think? (I see it briefly as the page loads, but I think that's expected behavior.) But the Radar hide is still behaving the same - toggling it off and on again hides, but it comes back on page refresh.

**Comment by nautiscarader** [comment]
Nope, for me it's the same. Also running 6.0.9, and both Radar and Reccomended appear upon reload. 

FYI if you are using Ublock, hide them with www.tumblr.com##aside

**Comment by nautiscarader** [comment]
Okay, so the avatars are sliding AGAIN!

**Comment by lihatt** [comment]
For larfs I tried disabling uBlock to see if maybe I was blocking Recommended by accident through that and not via Xkit (I had not deliberately added the suggested filter, but maybe something else was unintentionally causing it to hide), but no. Xkit seems to be doing it on its own for me. I see it start to load and then the page element disappears, leaving only Radar. (Although that disable/refresh/enable cycle sometimes had the odd side effect of Xkit itself apparently failing to load.) Chrome 84.0.4147.89 64-bit, Win10, if that makes any difference...

**Comment by nautiscarader** [comment]
Okay, the avatars are staying in place now... 

As for the radar/reccomended, disabling ublock does nothing. the two still appear, if I select and unselect option, they disappear, reappear after reload.

**Comment by lihatt** [comment]
I wasn't suggesting that would make it work, I was just trying to verify that it was _Xkit_ that was effecting the blocking and that I didn't somehow have it coming from uBlock without knowing it. It is strange that Xkit appears to be  behaving differently for the two of us.

**Comment by lihatt** [comment]
Seems to be working consistently for me now in 6.0.10, although I note the remarks in the two closed issues above that it may be dependent on exactly what order things get loaded in. (I guess that sort of situation might explain why Tag Tracking+'s feature of redirecting the tag search redirecting to /tagged/, which generally still works, occasionally seems to fail?)

**Comment by AprilSylph** [comment]
@lihatt Tag Tracking+ hasn't been updated for the new dashboard at all; you probably mean Classic Search, which is prone to the header-reset bug (#1869)

**Comment by lihatt** [comment]
I definitely mean TT+. I don't have Classic Search installed. I know TT+ hasn't been updated (I keep an eye on #1869, and boy do I miss tags in the sidebar) but nevertheless the feature to redirect /search/ to /tagged/ still functions (most of the time).

**Comment by AprilSylph** [comment]
Ah, you're talking about native tracked tags. On the new dash, those always link to /tagged/ pages - XKit doesn't modify the dropdown on the search bar on the new dash at all

**Comment by lihatt** [comment]
I don't want to derail this issue with too many unrelated comments, but I only noticed at all because, with the search dropdown currently my only option for keeping track of followed tags (other than bookmarking them all or something), I was visually startled a couple of times when I got the /search/ display instead of /tagged/ when clicking on a tracked tag from there. And sure enough that's what all their urls said. Refreshing the page put them back to /tagged/. So I assumed TT+ was still able to interfere with that bit at least and perhaps just hadn't loaded in time. If it's impossible for that to be the case then I have no idea what would cause it to sometimes get the urls formed as /search/ .
